tests/integration/yatai_server/test_bento_repository_api.py

```diff
@@ -142,24 +110,19 @@ def test_get(yatai_service_url, example_bento_service_class):
     assert svc_pb.bento_service_metadata.version == svc.version
 
 
-def test_list(yatai_service_url, example_bento_service_class):
-    example_bento_service_class = bentoml.ver(major=2, minor=5)(
-        example_bento_service_class
-    )
+def test_list_by_name(yatai_service_url, example_bento_service_class):
+    class ListByNameSvc(bentoml.BentoService):
+        pass
+
     yc = get_yatai_client(yatai_service_url)
-    test_model = TestModel()
-    svc = example_bento_service_class()
-    svc.pack('model', test_model)
+    svc = ListByNameSvc()
     svc.save(yatai_url=yatai_service_url)
 
     bentos = yc.repository.list(bento_name=svc.name)
-    assert len(bentos) == 7
+    assert len(bentos) == 1
```

Comment:
Pytest does not guarantee the order it runs these tests - that's why it was flaky @yubozhao